### PR TITLE
speech: add missing import for code sample

### PR DESCRIPTION
**Pull request by anguillanneuf**

### CODE REVIEW

File: speech/cloud-client/transcribe_enhanced_model.py [hunk 1]
```diff
@@ -28,7 +28,10 @@
 def transcribe_file_with_enhanced_model(path):
     """Transcribe the given audio file using an enhanced model."""
     # [START speech_transcribe_enhanced_model]
+    import io
```
Comment (nnegrey) [inline speech/cloud-client/transcribe_enhanced_model.py]: This import exists at the top of the file.

Comment (nnegrey) [reply speech/cloud-client/transcribe_enhanced_model.py]: @beccasaurus Should we also move standard imports inside the functions for imports or just leave them outside.

Comment (beccasaurus) [reply speech/cloud-client/transcribe_enhanced_model.py]: Ones like IO for reading files are 👍  – folks will find it helpful so things don't blow up

Comment (nnegrey) [reply speech/cloud-client/transcribe_enhanced_model.py]: Sounds good. @anguillanneuf
I'd say we should just make sure we remove the import above then and replace it inside the function where needed. :)

**Pull request merged**